packages/icon/README.md

```diff
@@ -0,0 +1,56 @@
+# Icon
+
+Mineral UI `Icon` component. Find out more about [Mineral UI](https://github.com/mineral-ui/mineral-ui).
+
+This package provides a large number of prepackaged icon components as well as
+a generic Icon component to which you can provide your own SVG.
+
+
+## Installation
+
+1. Install project [peer dependencies](../../docs/peer-dependencies.md), if not already installed
+
+2. Install the package
+
+  ```sh
+  npm install --save @mineral-ui/icon
+  ```
+
+
+## Usage
+
+### Prepackaged Icons
+
+```jsx
+import React from 'react';
+import IconHelp from '@mineral-ui/icon/components/IconHelp';
```

Comment:
We would have to copy the generated components to the root of dist.  It would make importing nicer for sure.  It might break some paths though, unless we also update them during the build step.  Converting it to something like lib would be easier.